src/utils/helpers.js

```diff
@@ -290,6 +290,13 @@ export function getURLSearchParams(intl = undefined) {
   const observationYear = urlSearchParams.get('observationYear')?.toLowerCase()
     || locals?.[bsoLocalAffiliation]?.observationYear
     || process.env.REACT_APP_LAST_OBSERVATION;
+  const idTypes = ['doi'];
+  const useHalId = !(
+    urlSearchParams.get('useHalId')?.toLowerCase() === 'false'
```

Comment:
Shoud we add this newly available query param into the doc on the localVariation page ?